### PR TITLE
Update readme.md

### PR DESCRIPTION
**Pull request by JordanHaigh**
Added sections on how to run program and required packages

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,55 @@
...
+* [Google Golang](http://sox.sourceforge.net/) -  Google's Go Language
+    * `sudo apt-get install golang`
+    * Note: You will need to set the GOPATH
+* [FFMPEG](https://ffmpeg.zeranoe.com/) - Decode, encode, filter and play files
+    * `sudo apt-get install ffmpeg`
+
+## How to run ##
+* Note: If you have not set your GOPATH correctly, it will not be able to perform the `go get` commmands
+
+Aqcuire repos:
```
Comment (benjojo) [inline README.md]: Acquire :)

File: README.md [hunk 2]
```diff
@@ -1,3 +1,55 @@
...
+If you encounter an error with Sox, saying that the "filter frequency must be less than sample-rate / 2", this might be an indication that the wav file is corrupt or too small.
+
+The script file will complete, creating new files, most importantly the 'encoded-bassline.wav'.
+
+Move to the decode folder
+```
+cd decode
+go build
+ffmpeg -i ../encoded-bassline.wav -f f64le -ar 44100 -ac 1 -y in.f64.data
+cat in.f64.data | tee sound | ./decode -input /dev/stdin
```
Comment (benjojo) [inline README.md]: the `sound` part is actually just a demo trick, If you want the `tee sound` thing to work, you need to also run 

```
$ mkfifo sound
$ ffplay -f f64le -ar 44100 -ac 1 -i sound &
```

before you do the final `cat`

**Review by benjojo — CHANGES_REQUESTED** [review]

**Comment by benjojo** [comment]
Thanks for pulling this together, there are a few little things, but other than that! thanks for the docs!

**Comment by JordanHaigh** [comment]
Fixed :)

**Comment by benjojo** [comment]
Thanks!

**Pull request merged**